src/exercise/06.js

```diff
@@ -4,26 +4,28 @@
 import * as React from 'react'
 
 function UsernameForm({onSubmitUsername}) {
-  // 🐨 add a submit event handler here (`handleSubmit`).
-  // 💰 Make sure to accept the `event` as an argument and call
-  // `event.preventDefault()` to prevent the default behavior of form submit
-  // events (which refreshes the page).
-  //
-  // 🐨 get the value from the username input (using whichever method
-  // you prefer from the options mentioned in the instructions)
-  // 💰 For example: event.target.elements[0].value
-  // 🐨 Call `onSubmitUsername` with the value of the input
-
-  // 🐨 add the onSubmit handler to the <form> below
-
-  // 🐨 make sure to associate the label to the input.
-  // to do so, set the value of 'htmlFor' prop of the label to the id of input
+
+  const [inputValue, setInputValue] = React.useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    onSubmitUsername(event.target.elements.username.value);
+  }
+
+  function handleChange(event) {
+    const value = event.target.value;
+    setInputValue(value.toLowerCase());
+  }
+
   return (
-    <form>
+    <form
+      onSubmit={handleSubmit}
+    >
       <div>
-        <label>Username:</label>
-        <input type="text" />
+        <label htmlFor='username'>Username:</label>
```

Comment:
Our ESLint standards would tell you to use double quotes rather than single quotes here, for consistency in the codebase as well as similarity to writing normal HTML:
```suggestion
        <label htmlFor="username">Username:</label>
```